be/src/exprs/json_functions.cpp

```diff
@@ -147,7 +147,7 @@ Status JsonFunctions::extract_from_object(simdjson::ondemand::object& obj, const
     // Skip the first $.
     for (int i = 1; i < jsonpath.size(); i++) {
         if (UNLIKELY(!jsonpath[i].is_valid)) {
-            return Status::InvalidArgument(fmt::format("invalid json path: {}", jsonpath[i].key));
+            return Status::InvalidArgument(fmt::format("invalid json path: {}", jsonpaths_to_string(jsonpath)));
```

Comment:
why do you print the whole jsonpath rather than the exect error part?
that is : if we cant find the value of key `$.a.b`, don't print the whole jsonpath `$.a.b.c.d`